### ISSUE TITLE
Power button disappeared from splash screen.

### ISSUE THREAD

**Issue by LazaroFilm** [jordanruthe/KlipperScreen#552]
The power button disappeared from the splash screen next to restart /firmware restart etc... I can still power the printer from the power menu but it's a handful of extra taps. ![image](https://user-images.githubusercontent.com/18351100/159122608-bd03bcdf-045e-4517-9c40-c1a27515ddb2.jpeg)

**Comment by alfrix** [comment]
https://klipperscreen.readthedocs.io/en/latest/Configuration/#printer-options
you need to add the moonraker power_device (or devices) that turn on the printer
example of klipperscreen.conf
```
[main]
show_cursor: False

[printer Voron 2.4]
# This power device is the 'associated power device' that will turn on the printer in the splash screen
power_devices: PSU
```

**Comment by alfrix** [comment]
There was an issue affecting this, should be fixed with 1dee91456493ea659f2ee637c57450141a0278c2
